config/locales/two_factor_authentication/en.yml

```diff
@@ -133,7 +133,8 @@ en:
       less_secure_label: Less secure
       more_secure_label: More Secure
       phone: Text or Voice Message
-      phone_info: Receive a secure code by (SMS) text or phone call to your device.
+      phone_info_html: Receive a secure code by (SMS) text or phone call. <strong>You
+        need to select another method in addition to this one.</strong>
```

Comment:
If this is feature-flagged, do we need to make sure we only include the bolded message if the feature flag is enabled?